app/main/views/users.py

```diff
@@ -63,6 +63,12 @@ def list_users():
             links={}
         )
 
+    only_buyers = request.args.get('only_buyers')
```

Comment:
I said this yesterday in person, but there was a moment a last year when @minglis (our old  technical architect) said that "APIs always get bigger -- never smaller".  
He was arguing against generalising something that we didn't need to be generalised yet. 

I think keep this `only_buyers` switch instead of passing in a role.  Right now, `buyers` are the only kinds of users we're looking to bulk export for this story.